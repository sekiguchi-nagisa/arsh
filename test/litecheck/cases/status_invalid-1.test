#!/bin/sh
echo hello
exit 0
# CHECK: hello
# STATUS: 0.0