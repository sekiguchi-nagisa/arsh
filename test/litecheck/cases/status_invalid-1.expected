status_invalid-1.test:5: [error] STATUS directive must be positive number: `0.0'